databox/core/register.ts

```diff
@@ -0,0 +1,3 @@
+import DataboxForPuppeteerRunner from './index';
```

Comment:
This import is misnamed, but I think this file is unnecessary. Databox-Core should always be registered in server.

Reply:
After full review, I now understand this has significant nuances. You're really saying register the databox-client modules. The structure of having this also be the place to host run and upload apis makes the idea of "registering" this awkward though.